### PR TITLE
[DUOS-1902][risk=no] Use DAR Collection on Admin Console Page

### PR DESCRIPTION
**Pull request by connorlbark**
## Addresses

https://broadworkbench.atlassian.net/browse/DUOS-1902

----
Have you read [Terra's Contributing Guide](https://github.com/DataBiosphere/terra-ui/wiki/Contributor-Guide) lately? If not, do that first.

- Label PR with a Jira ticket number and include a link to the ticket
- Label PR with a security risk modifier [no, low, medium, high]
- PR describes scope of changes
- Get a minimum of one thumbs worth of review, preferably two if enough team members are available
- Get PO sign-off for all non-trivial UI or workflow changes
- Verify all tests go green
- Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: cypress/component/DarCollectionTable/admin_actions.spec.js [hunk 1]
```diff
@@ -1,49 +1,71 @@
...
+const darColl = {
+  'darCollectionId': 2345,
+  "referenceIds": [
+    '4a3fd-g77fd-2f345-4h2g31',
+    '0a4jn-g838d-bsdg8-6s7fs7',
+  ],
+  'darCode': "DAR-9583",
+  'name': "Example DAR 1",
+  "submissionDate": "2022-07-26",
+  'researcherName': "John Doe",
```
Comment (rushtong) [inline cypress/component/DarCollectionTable/admin_actions.spec.js:16]: Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/DataBiosphere/duos-ui/pullRequest?prid=9874341)
```suggestion
  'researcherName': 'John Doe',
```

File: cypress/component/DarCollectionTable/admin_actions.spec.js [hunk 2]
```diff
@@ -1,49 +1,71 @@
...
+  'institutionName': "Broad Institute",
+  "status": "Draft",
+  "actions": [
```
Comment (rushtong) [inline cypress/component/DarCollectionTable/admin_actions.spec.js:19]: Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/DataBiosphere/duos-ui/pullRequest?prid=9874341)
```suggestion
  'actions': [
```

File: cypress/component/DarCollectionTable/admin_actions.spec.js [hunk 3]
```diff
@@ -1,49 +1,71 @@
...
+  'darCollectionId': 2345,
+  "referenceIds": [
+    '4a3fd-g77fd-2f345-4h2g31',
+    '0a4jn-g838d-bsdg8-6s7fs7',
+  ],
+  'darCode': "DAR-9583",
+  'name': "Example DAR 1",
+  "submissionDate": "2022-07-26",
+  'researcherName': "John Doe",
+  'institutionName': "Broad Institute",
```
Comment (rushtong) [inline cypress/component/DarCollectionTable/admin_actions.spec.js:17]: Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/DataBiosphere/duos-ui/pullRequest?prid=9874341)
```suggestion
  'institutionName': 'Broad Institute',
```

File: cypress/component/DarCollectionTable/admin_actions.spec.js [hunk 4]
```diff
@@ -1,49 +1,71 @@
...
-  updateCollections: () => {},
+const darColl = {
+  'darCollectionId': 2345,
+  "referenceIds": [
+    '4a3fd-g77fd-2f345-4h2g31',
+    '0a4jn-g838d-bsdg8-6s7fs7',
+  ],
+  'darCode': "DAR-9583",
+  'name': "Example DAR 1",
+  "submissionDate": "2022-07-26",
```
Comment (rushtong) [inline cypress/component/DarCollectionTable/admin_actions.spec.js:15]: Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/DataBiosphere/duos-ui/pullRequest?prid=9874341)
```suggestion
  'submissionDate': "2022-07-26",
```

File: cypress/component/DarCollectionTable/admin_actions.spec.js [hunk 5]
```diff
@@ -1,49 +1,71 @@
...
-          2: {status: "Closed"}
-        }
-      }
-    }
-  },
-  showCancelModal: () => {},
-  updateCollections: () => {},
+const darColl = {
+  'darCollectionId': 2345,
+  "referenceIds": [
```
Comment (rushtong) [inline cypress/component/DarCollectionTable/admin_actions.spec.js:9]: Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/DataBiosphere/duos-ui/pullRequest?prid=9874341)
```suggestion
  'referenceIds': [
```

File: cypress/component/DarCollectionTable/admin_actions.spec.js [hunk 6]
```diff
@@ -1,49 +1,71 @@
...
+  'name': "Example DAR 1",
+  "submissionDate": "2022-07-26",
+  'researcherName': "John Doe",
+  'institutionName': "Broad Institute",
+  "status": "Draft",
+  "actions": [
+    'Review',
+    'Cancel',
+  ],
+  "hasVoted": false,
```
Comment (rushtong) [inline cypress/component/DarCollectionTable/admin_actions.spec.js:23]: Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/DataBiosphere/duos-ui/pullRequest?prid=9874341)
```suggestion
  'hasVoted': false,
```

File: cypress/component/DarCollectionTable/admin_actions.spec.js [hunk 7]
```diff
@@ -1,49 +1,71 @@
...
-  showCancelModal: () => {},
-  updateCollections: () => {},
+const darColl = {
+  'darCollectionId': 2345,
+  "referenceIds": [
+    '4a3fd-g77fd-2f345-4h2g31',
+    '0a4jn-g838d-bsdg8-6s7fs7',
+  ],
+  'darCode': "DAR-9583",
+  'name': "Example DAR 1",
```
Comment (rushtong) [inline cypress/component/DarCollectionTable/admin_actions.spec.js:14]: Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/DataBiosphere/duos-ui/pullRequest?prid=9874341)
```suggestion
  'name': 'Example DAR 1',
```

File: cypress/component/DarCollectionTable/admin_actions.spec.js [hunk 8]
```diff
@@ -1,49 +1,71 @@
...
-  updateCollections: () => {},
+const darColl = {
+  'darCollectionId': 2345,
+  'referenceIds': [
+    '4a3fd-g77fd-2f345-4h2g31',
+    '0a4jn-g838d-bsdg8-6s7fs7',
+  ],
+  'darCode': "DAR-9583",
+  'name': 'Example DAR 1',
+  'submissionDate': "2022-07-26",
```
Comment (rushtong) [inline cypress/component/DarCollectionTable/admin_actions.spec.js:15]: Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/DataBiosphere/duos-ui/pullRequest?prid=9874341)
```suggestion
  'submissionDate': '2022-07-26',
```

**Comment by lu-c** [comment]
This ah, looks good to me, but it conflicts with 1904 - I just pulled it directly into that branch, and it seems to be fine!

**Review by lu-c — APPROVED** [review]

**Review by shaemarks — APPROVED** [review]
Looks good, but with these changes already pulled into #1710 I don't think it's necessary to merge here

**Comment by JVThomas** [comment]
Closing. Changes are incorporated in https://github.com/DataBiosphere/duos-ui/pull/1699

**Pull request closed without merging**